src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir;
+  src: url("/fonts/Avenir.ttc"),
+    url("/fonts/Avenir-Book.ttf"),
+    url("/fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.wrapper {
+  position: relative;
+  margin-top: 20px;
+}
+
+.input {
+  width: 100%;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  background: #fff;
+  border-radius: 4px;
+  font-weight: 900;
+  color: #3d4e61;
+  font-family: inherit;
+}
+
+.input::placeholder {
+  font-weight: 300;
+}
+
+.big__input {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.small__input {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+  height: 42px;
+}
```

Comment:
you should have default search-bar and big/small. Not big and small at the same time. One of them is just search-bar without big/small word.